api/server/router/swarm/cluster_routes.go

```diff
@@ -132,7 +132,7 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 
 	id, err := sr.backend.CreateService(service, encodedAuth)
 	if err != nil {
-		logrus.Errorf("Error creating service %s: %v", id, err)
+		logrus.Errorf("Error creating service: %v", err)
```

Comment:
If createService fails, and user does not specify, it will be empty. I think the chance is really high, but not sure whether it is larger than non-empty. It depends on user's action.